psychiatric_guide_app/settings.py

```diff
@@ -143,8 +143,9 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = 'static'
+STATIC_ROOT = ''
+STATICFILES_DIRS = ( os.path.join('static'), )
 LOGIN_REDIRECT_URL = 'backend-home'  # Sets redirect after login
 CRISPY_TEMPLATE_PACK = 'bootstrap4'
 LOGIN_URL = 'login-view' # Redirect for the login page for @login_required
-# LOGOUT_URL = 'logout-view' # Redirects to login page after logout
\ No newline at end of file
+# LOGOUT_URL = 'logout-view' # Redirects to login page after logout
```

Comment:
We should just get rid of this since it is commented out. The log out button works as intended currently without the need for a LOGOUT_URL constant.